Package.swift

```diff
@@ -22,13 +22,16 @@ let package = Package(
     products: [
         .library(
             name: "DeepLearning",
-            targets: ["DeepLearning"]),
+            targets: ["DeepLearning"/*, "third_party"*/]),
     ],
     dependencies: [],
     targets: [
         .target(
             name: "DeepLearning",
             dependencies: []),
+        // .target(
```

Comment:
You can specify paths for individual targets. See example [here in the swift-models repo](https://github.com/tensorflow/swift-models/blob/4275c36b8ea229c761a3d5d339baf674dbb148cd/Package.swift#L23).